packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.controller.js

```diff
@@ -0,0 +1,40 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from './loadbalancers.constants';
+
+export default class OctaviaLoadBalancersCtrl {
+  /* @ngInject */
+  constructor(atInternet, Alerter, $translate) {
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
```

Comment:
Alerter and $translate don't seem to be used, I think you can removed them

Reply:
Done